src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -1247,9 +1298,15 @@ TEST_F(ConnectionHandlerTest,
   unprotected_services_.push_back(kControl);
   SetSpecificServices();
   // Start new session with Audio service
```

Comment:
@tkireva Please move the comment to the right place, probably leaving an additional one describing the situation would be nice